examples/java/src/main/java/org/apache/beam/examples/complete/cdap/CdapMain.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.examples.complete.cdap;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cdap.cdap.api.plugin.PluginConfig;
+import io.cdap.plugin.salesforce.SalesforceConstants;
+import io.cdap.plugin.salesforce.plugin.source.batch.SalesforceSourceConfig;
+import io.cdap.plugin.salesforce.plugin.source.batch.util.SalesforceSourceConstants;
+import org.apache.beam.examples.complete.kafkatopubsub.KafkaToPubsub;
+import org.apache.beam.examples.complete.kafkatopubsub.options.KafkaToPubsubOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class CdapMain {
+
+  /* Logger for class.*/
+  private static final Logger LOG = LoggerFactory.getLogger(CdapMain.class);
+
+  public static void main(String[] args) {
+    if (args.length == 0) {
+      LOG.error("Need to pass the json file path as a first arg");
+      return;
+    }
+    String jsonFilePath = args[0];
+
+    try {
+      TypeReference<HashMap<String, Object>> typeRef = new TypeReference<HashMap<String, Object>>() {
+      };
+      Map<String, Object> mapping = new ObjectMapper()
+          .readValue(new File(jsonFilePath), typeRef);
+
+      //1
```

Comment:
Could you please move each unit test to the separate unit test method rather than having it the way it is now? And could you please have more meaningful comments? It doesn't seem clear

Reply:
Also, we can provide additional comments to separate code into three blocks: Arrange, Act and Assert

Reply:
Moved to the `ConfigWrapperTest` class.